tests/install/steps/test_expansion.py

```diff
@@ -0,0 +1,61 @@
+import pathlib
+import string
+
+from pytest_bdd import scenario, then, parsers, when
+import testinfra
+import kubernetes as k8s
+import yaml
+
+# Scenarios
+@scenario('../features/expansion.feature', 'Add one node to the cluster')
+def test_cluster_expansion(host):
+    pass
+
+# When {{{
+
+@when(parsers.parse('we declare a new node on host "{hostname}"'))
+def declare_node(host, request, kubeconfig, hostname):
+    """Declare the given node in Kubernetes."""
+    ssh_config = request.config.getoption('--ssh-config')
+    node_ip = get_node_ip(hostname, ssh_config)
+    node_manifest = get_node_manifest(node_ip)
+    k8s.config.load_kube_config(config_file=kubeconfig)
+    k8s_client = k8s.client.CoreV1Api()
+    k8s_client.create_node(body=node_from_manifest(node_manifest))
+
+# }}}
+# Then {{{
+
+@then(parsers.parse('node "{hostname}" is registered in Kubernetes'))
+def check_node_is_registered(host, request, kubeconfig, hostname):
```

Comment:
You don't need `host` or `request` here:

```suggestion
def check_node_is_registered(kubeconfig, hostname):
```

Reply:
Indeed.

I thought `host` was mandarory because I saw this in another file:

```
# Scenarios
@scenario('../features/dns_resolution.feature', 'check DNS')
def test_dns(host):
    pass
```

But I guess I can remove it from the `@scenario` in my file as well, no?